CanvasApi.Client/EnrollmentTerms/Models/IEnrollmentTermListOptions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using CanvasApi.Client.EnrollmentTerms.Enums;
+using Newtonsoft.Json;
+
+namespace CanvasApi.Client.EnrollmentTerms
+{
+    public interface IEnrollmentTermListOptions
+    {
+        /// <summary>
+        /// Array of additional information to include.and “submission” options are included. The “assignment_visibility” option additionally requires that the Differentiated Assignments course feature be turned on. If “observed_users” is passed along with “assignments” and “submission”, submissions for observed users will also be included as an array.
+        ///
+        /// Allowed values: <see cref="EnrollmentTermIncludes"/>
+        /// overrides
+        /// </summary>
+        [JsonProperty("include")] IEnumerable<EnrollmentTermIncludes> Include { get; set; }
```

Comment:
Looks like the Enum is actually called "EnrollmentTermListIncludes". Looks like you might be missing the "List" in the name.

Reply:
build server is rejected this file.+